### ISSUE TITLE
ERR200 문의

### ISSUE THREAD

**Issue by minseungki** [kginicis-ts/INICIS_Cancel#15]

**Comment by minseungki** [comment]
![image](https://github.com/kginicis-ts/INICIS_Cancel/assets/33819613/b5862c0b-40fa-4d45-b8d1-be08179752a4)

혹시 이 조회 버튼 누르는게 맞나요?

**Comment by ts-inicis** [comment]
> ![image](https://user-images.githubusercontent.com/33819613/244272030-b5862c0b-40fa-4d45-b8d1-be08179752a4.png)
> 
> 혹시 이 조회 버튼 누르는게 맞나요?

네 맞습니다.

**Comment by ts-inicis** [comment]
네 맞습니다.

**Comment by minseungki** [comment]
![image](https://github.com/kginicis-ts/INICIS_Cancel/assets/33819613/7f55f40a-626e-4059-8f97-fea7d95a7cfc)

만약에 이렇게 생성 후에 누군가가 갱신버튼을 눌러서 key와 iv값을 변경하면 서비스에 장애가 발생하겠죠?

**Comment by ts-inicis** [comment]
네,  고유한 키값이기 때문에 기존 정상 설정 완료된 API 구성 이후에 KEY , IV 값을 갱신한다면 갱신된 키값을 설정해 주셔야 합니다.

**Comment by minseungki** [comment]
넵 빠른 답변 감사합니다! @
